### ISSUE TITLE
Projects on Windows not working

### ISSUE THREAD

**Issue by P1P0S** [nvimdev/dashboard-nvim#347]
Basically the 'recent projects' feature is not working on Windows, (I'm not an experienced programmer) so from what I've looked into it, there seems to be some issue with how the cache file is being opened for writing, in the 'utils.lua' file.

![image](https://user-images.githubusercontent.com/62353660/233497124-bd9d5ebc-c3f0-484d-bbe1-8cf86117f167.png)

I solved my problem by doing a simple 'if' statement to check if the 'os' is Windows. Then, it opens the cache file for writing using 'io.open'.

![image](https://user-images.githubusercontent.com/62353660/233497943-bede91ca-23a5-4dac-83f2-f18fbb5931f6.png)

Well, now it's writing the project directories to the cache file.

![image](https://user-images.githubusercontent.com/62353660/233498419-5e14352f-0623-45f6-91ad-db339257dbd3.png)

**Comment by glepnir** [comment]
first there has system check in util and write cache is in luv callback is async . no reason use a block io.open. I don't use windows. someone can fix this pr welcome.

**Comment by jacknicklenson** [comment]
Yes, I have this issue as well on windows 11. @P1P0S if you solved,  can you create PR ?

**Comment by tidaeu** [comment]
I'm running into this as well. :(
